packages/vite/package.json

```
@@ -29,12 +40,12 @@
   },
   "homepage": "https://github.com/vitejs/vite/tree/main/#readme",
   "scripts": {
-    "dev": "rimraf dist && rollup -c -w",
+    "dev": "rimraf dist && pnpm run build-bundle -w",
     "build": "rimraf dist && run-s build-bundle build-types",
-    "build-bundle": "rollup -c",
+    "build-bundle": "rollup --config rollup.config.ts --configPlugin esbuild",
```

```suggestion
    "build-bundle": "rollup --config rollup.config.ts --configPlugin typescript",
```

Could we reuse `@rollup/plugin-typescript` so we don't have to bring in a new `rollup-plugin-esbuild` dependency?

---

I failed to make it work, as we don't have package level `tsconfig.json` (while if you add it, it would cause other issues). If you can make it working, feel free to directly push it to this branch, otherwise I guess we could leave it for future PRs.